app/services/data/get-full-overview.js

```diff
@@ -0,0 +1,39 @@
+const knex = require('../../../knex').web
+const orgUnitFinder = require('../helpers/org-unit-finder')
+const orgUnitConstants = require('../../constants/organisation-unit')
+
+module.exports = function (id, type) {
+  var orgUnit = orgUnitFinder('name', type)
+  var table = 'individual_case_overview'
+  var whereClause = ''
+  var orderBy = 'ldu_name, team_name'
```

Comment:
In the feedback email its says order by organisational level, so if this is at the region level, should it not be ordered by the division name?

Reply:
This is added on line 29 if the organisation level is region or above